packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -293,62 +313,95 @@ export default class Import extends Component {
                           <div>
                             <hr />
-                            <div className='alert alert-info' role='alert'>
-                              <strong>{_('vmImportFileType', { type })}</strong> {_('vmImportConfigAlert')}
+                            <div className='alert alert-danger' role='alert'>
+                              <strong>{_('vmImportError')}</strong>{' '}
+                              {(error && error.message) || _('noVmImportErrorDescription')}
                             </div>
-                            <VmData {...data} ref={`vm-data-${vmIndex}`} pool={pool} />
-                          </div>
-                        )
-                      ) : (
-                        <div>
-                          <hr />
-                          <div className='alert alert-danger' role='alert'>
-                            <strong>{_('vmImportError')}</strong>{' '}
-                            {(error && error.message) || _('noVmImportErrorDescription')}
                           </div>
-                        </div>
-                      )}
-                    </div>
-                  ))}
+                        )}
+                      </div>
+                    ))}
+                  </div>
+                ) : (
+                  <p>{_('noSelectedVms')}</p>
+                )}
+                <hr />
+                <div className='form-group pull-right'>
+                  <ActionButton
+                    btnStyle='primary'
+                    disabled={!vms.length}
+                    className='mr-1'
+                    form='import-form'
+                    handler={this._import}
+                    icon='import'
+                    redirectOnSuccess={getRedirectionUrl}
+                    type='submit'
+                  >
+                    {_('newImport')}
+                  </ActionButton>
+                  <Button onClick={this._handleCleanSelectedVms}>{_('importVmsCleanList')}</Button>
                 </div>
-              ) : (
-                <p>{_('noSelectedVms')}</p>
-              )}
-              <hr />
-              <div className='form-group pull-right'>
+              </div>
+            ) : (
+              <div>
+                <FormGrid.Row>
+                  <FormGrid.LabelCol>{_('url')}:</FormGrid.LabelCol>
+                  <FormGrid.InputCol>
+                    <Input
+                      className='form-control'
+                      onChange={this.linkState('url')}
+                      placeholder='https://my-company.net/vm.xva'
+                      type='url'
+                    />
+                  </FormGrid.InputCol>
+                </FormGrid.Row>
+                <FormGrid.Row>
+                  <FormGrid.LabelCol>{_('fileType')}:</FormGrid.LabelCol>
+                  <FormGrid.InputCol>
+                    <Input className='form-control' onChange={this.linkState('type')} placeholder='xva' />
```

Comment:
This should be a select so that the user can only choose supported formats. And since we only support XVA for now, it should only contain XVA, selected by default.